build-packages/eslint-plugin-scandipwa-guidelines/lib/rules/forbid-prop-types-in-shape.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @fileoverview Forbid usage of empty PropTypes.shape
+ * @author Tatiana Karamorina
+ */
+
+module.exports = {
+    meta: {
+        docs: {
+            description: 'Forbid usage of empty PropTypes.shape.',
+            category: 'Coding standard',
+            recommended: true
+        }
+    },
+    create: (context) => ({
+        MemberExpression(node) {
+            function isShapePropType(node) {
+                return node.object.name === 'PropTypes' && node.property.name === 'shape';
+            }
+
+            if (isShapePropType(node)) {
+                // forbid `PropTypes.shape`
+                if (node.parent.arguments === undefined) {
+                    context.report({
+                        node: node.property,
+                        message: `Found empty PropTypes.shape`
```

Comment:
Please give a more detailed description like "Empty PropTypes.shape is forbidden ...".
Now i see that PropTypes.shape is found without the reason why i have an error.